packages/e2e-test-utils/src/insert-and-populate-block.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	insertBlock,
+	textContentAreas,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Inserts a content block and then, if it has text content areas, fills them with text
+ *
+ * @param {string} blockName The type of block to insert
+ * @param {string} content  The text to enter into each contenteditable area
+ * @return {Promise} A promise that resolves when all the blocks are inserted and filled with content.
+ */
+export async function insertAndPopulateBlock( blockName, content ) {
+	await insertBlock( blockName );
+	// typing populates the first content area
+	await page.keyboard.type( content );
+
+	// if there are more contenteditable elements, select and populate them too:
+	const blocks = await textContentAreas( { empty: true } );
```

Comment:
The name of the variable is a bit confusing, how about:
```suggestion
	const blockContentAreas = await textContentAreas( { empty: true } );
```